pkg/registry/generic/etcd/etcd_test.go

```diff
@@ -339,6 +339,53 @@ func TestEtcdUpdate(t *testing.T) {
 
 }
 
+func TestNoOpUpdates(t *testing.T) {
+	server, registry := NewTestGenericEtcdRegistry(t)
```

Comment:
Sorry - I'm afraid I don't understand:
- what do you mean by "inserts an old incorrect record (containing resourceVersion) into etcd"?
- actual updates are tested by a bunch of other tests, I don't think we need to test it here again